birdy/dependencies.py

```diff
@@ -11,7 +11,7 @@
 import warnings
 from .exceptions import IPythonWarning
 
-warnings.filterwarnings('default', category=IPythonWarning)
```

Comment:
Wouldn't this make them disabled in every situation? 
No more point to raise the warning?

Maybe wrap the cli code in a `with warnings.filterwarnings()` block to disable them where they need to be?

Reply:
Yes, `ignore` would just disable the warnings. The imports are only needed when birdy is used in a notebook.

Reply:
Is there a way to "detect" that we are in a notebook? The default/ignore could be set accordingly.

Reply:
If we are in a notebook and these modules are not available, then the code will fail. I think the simplest solution is just to make sure that the error message when this happens is clear enough. So yes, I think we could remove these try statements entirely, but I didn't have the time to follow through.